rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -195,6 +208,8 @@ docker run --name $docker_name \
   $nvidia_devices \
   --device=/dev/fuse \
   --security-opt apparmor:unconfined \
+  --volume /tmp/pai-root/bootstrap/$APP_ID/$CONTAINER_ID:/pai/bootstrap \
+  --volume /tmp/pai-root/code/$APP_ID/$CONTAINER_ID:/pai/code \
   --volume /var/drivers/nvidia/current:/usr/local/nvidia:ro \
```

Comment:
Is it possible to mount to "WORK_DIR" directly?

Reply:
no, user may store some files under WORK_DIR in their Docker image, mounting WORK_DIR will overwrite the whole directory.